WEATHERAPP/Components/SignUp/SignUp.html

```diff
@@ -1,2 +1,31 @@
 <!DOCTYPE html>
-<head>signup page of the weather app for new user</head>
\ No newline at end of file
+<html>
+    <head lang="en">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <meta charset="UTF-8">
+        <title>WEATHER APP</title>
+    <link rel="stylesheet" href="SignUp.css">
+    </head>
+    <body>
+        <div class="background">
+            <header class="app-header">WEATHER APP</header>
+            <main class="signup-form">
+                <section class="input-section">
+                    <label for="username" class="input-label">Username</label>
+                    <input type="text" class="input-fields" name="username" id="username" placeholder="Username" aria-label="username" required>
+                </section>
+                <section class="input-section">
+                    <label for="password" class="input-label">Password</label>
+                    <input type="password" class="input-fields" name="password" id="password" placeholder="Password" aria-label="password" required>
+                </section>
+                <section class="input-section">
+                    <label for="email" class="input-label">Email</label>
+                    <input type="email" class="input-fields" name="email" id="email" placeholder="Email" aria-label="email" required>
+                </section>
+                <section class="buttons-section">
+                    <button class="signup-button" id="sign-up">Sign UP</button>
```

Comment:
Specify type attribute for button (e.g., type="submit").

Reply:
Resolved